tests/test_utils.py

```diff
@@ -0,0 +1,10 @@
+from pytest_utils import mat_url  # noqa: F401
+
+
+def test__load_mat(mat_url):  # noqa: F811
```

Comment:
Use https://github.com/astropy/pytest-remotedata . Also no need for mangling.
```suggestion
@pytest.mark.remote_data
def test_load_mat(mat_url):
```